### DISCUSSION TITLE
Can I schedule observation of SourceAssets?

### DISCUSSION THREAD

**Discussion by wrb2** [dagster-io/dagster#11495] [Q&A]
For source assets that have observation function defined, I can run it using a button in Dagit.

Is there a way to schedule it, so it is observed every hour or something?

**Comment by sryza** [comment]
@smackesey - is there currently a way to do this?

**Reply by smackesey** [reply]
Hi @wrb2 apologies for my late reply-- there is currently no way to do this, but we are looking hard at observable source assets now and you can expect an update in the coming weeks.
